aws/chef-bootstrap.sh

```diff
@@ -106,13 +107,13 @@ if [ -f /etc/chef/client.pem ] ; then
 fi
 
 if [ "${LOCAL_MODE}" = "1" ]; then
-  mkdir -p /opt/chef-zero/{cookbooks,environments}
+  mkdir -p /var/chef/{cookbooks,environments}
```

Comment:
there's a bit of admittedly-confusing [heredoc](http://tldp.org/LDP/abs/html/here-docs.html) language-switching going on above: this file is a Bash shell script that writes content (via inline heredoc syntax) to a `.rb` Ruby DSL (`chef_repo_path` and `local_mode` are Chef [`client.rb`](https://docs.chef.io/config_rb_client.html) parameters), that will be loaded/executed as Ruby during the initial `chef-client` run. So it's not possible to use the inner Ruby DSL parameters directly in the outer Bash script. What I can do at least is add a variable to clean up the `/var/chef` repetition, which should help a bit.